variables/ageCalculator.js

```diff
@@ -1 +1,5 @@
-// Enter code below
\ No newline at end of file
+var birth=1987;
+var yyyy=2016;
+var nn=yyyy-birth;
+var nnn=nn+1;
+console.log('I will be either '+nn+' or '+nn++n+' in '+yyyy+'.');
```

Comment:
Maybe the instructions weren't very clear but you should consider using variable names that are more descriptive such as
```js
var birthYear = 1987;
var futureYear = 2025;
var futureAge = futureYear - birthYear;
console.log('I will be either ' + futureAge + ' or ' + (futureAge - 1) + ' in ' + futureYear + '.');
```